Source/ui_unix/openglwindow.cpp

```diff
@@ -0,0 +1,34 @@
+#include "openglwindow.h"
+#include <QResizeEvent>
+
+OpenGLWindow::OpenGLWindow(QWindow *parent)
+    : QWindow(parent)
+{
+    QSurfaceFormat format;
+    format.setVersion(3, 2);
+    format.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
Alright, after testing, I think the best way to go is to create the core context and keep the `glewExperimental` assignation. I've made changes to the way context creation is handled on Win32 to match that: 142513d877a1133b6dd674325aac5742a02daba9.